cluster-autoscaler/cloudprovider/alicloud/alicloud_cloud_config.go

```diff
@@ -41,7 +41,7 @@ func (cc *cloudConfig) isValid() bool {
 	}
 
 	if cc.AccessKeySecret == "" {
-		cc.AccessKeySecret = os.Getenv(accessKeyScret)
+		cc.AccessKeySecret = os.Getenv(accessKeySchwret)
```

Comment:
I did not notice that before. This acutally introduces another typo.

Reply:
So sorry for my mistake, it's caused by my squash ,I have fixed it